src/Sulu/Bundle/RouteBundle/Resources/js/containers/Form/fields/pageTreeRoute.scss

```diff
@@ -0,0 +1,13 @@
+.resource-locator-container {
```

Comment:
If we would reuse the `ResourceLocator` field type it would also not be necessary to duplicate this code.